src/components/UpdateTodo.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+<div class="todoBox">
+    <h3 class="title">TITLE</h3>
+    <input class="titleBox" v-model="updatedtitles" type="text">
+    <div v-if="updatedtitles.length == 0" class="error">* please enter title</div>
+    <h4 class="title">DETAILS</h4>
+    <textarea class="detailBox" v-model="updateddetails" rows="4" cols="30"></textarea>
+    <div v-if="updateddetails.length == 0" class="error">* please enter details</div>
+    <button class="updatedTodo" @click="updateProject(index)">Update todo</button>
+</div>
+</template>
+
+<script>
+export default {
+    name: 'UpdateTodo',
+    data() {
+        return {
+            updatedtitles: ``,
+            updateddetails: '',
```

Comment:
move both updatedtitles and updateddetails into a single object and use it from their.
SO it can be like
project: {
title: "",
description: ""
}